README.md

```diff
@@ -73,6 +61,10 @@ More information here: https://theforeman.org/manuals/1.22/quickstart_guide.html
 
 Write down the admin password that the installer outputs.
 
+## SELinux
+
+Foreman version 1.24 already includes a SELinux rule and assignment for port 19090 used in this document. When installing on older Foreman versions, put SELinux into permissive.
```

Comment:
I think we should wait for Foreman 1.24 and then tell people to install that.  This isn't meant to be a general guide for all kinds of Foreman versions, but only to document what is still missing from the current release.  At one point, we should just list the necessary forman-installer options here, for example.